usr/src/cmd/bc/bc.y

```diff
@@ -764,7 +764,7 @@ routput(int *p)
 	if (p >= &b_space[0] && p < &b_space[b_sp_max]) {
 		/* part of a bundle */
 		while (*p != 0)
-			routput((int *)*p++);
+			routput((int *)(uintptr_t)*p++);
```

Comment:
The contents of `p` here come from `b_space` doesn't that still leave this truncated, just the compiler not warning us?  We also, in `bundle()` seem to pull things that are pointers out of a `va_list` as if they were integers, and generally bundling seems like it must still be broken here.  Are we just looking out because these tend to be in the heap, and thus _usually_ fit in 32bits, or am I missing something?